cljfmt/src/cljfmt/core.cljc

```diff
@@ -336,6 +363,9 @@
 (defmethod indenter-fn :block [sym context [_ idx]]
   (fn [zloc] (block-indent zloc sym idx context)))
 
+(defmethod indenter-fn :thread [_ _ _]
+  (constantly nil))
+
```

Comment:
I think we need to rethink how the indentation functions are applied, as the abstraction has clearly failed if we have an indenter function that does nothing.